spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -73,56 +73,56 @@
-  context "replacing OAI harvesting with dor-fetcher" do
-      it "has a dor-fetcher client" do
-        expect(@indexer.dor_fetcher_client).to be_an_instance_of(DorFetcher::Client)
-      end
+  context 'replacing OAI harvesting with dor-fetcher' do
+    it 'has a dor-fetcher client' do
+      expect(@indexer.dor_fetcher_client).to be_an_instance_of(DorFetcher::Client)
+    end
 
-      it "druids method calls druid_array and get_collection methods on fetcher_client" do
-        VCR.use_cassette('get_collection_druids_call') do
-          expect(@indexer.resources.map(&:druid)).to match_array ["druid:yg867hg1375", "druid:jf275fd6276", "druid:nz353cp1092", "druid:tc552kq0798", "druid:th998nk0722", "druid:ww689vs6534"]
-        end
+    it 'druids method calls druid_array and get_collection methods on fetcher_client' do
+      VCR.use_cassette('get_collection_druids_call') do
+        expect(@indexer.resources.map(&:druid)).to match_array ['druid:yg867hg1375', 'druid:jf275fd6276', 'druid:nz353cp1092', 'druid:tc552kq0798', 'druid:th998nk0722', 'druid:ww689vs6534']
       end
+    end
 
-      it "gets the configuration of the dor-fetcher client from included yml file" do
-        expect(@indexer.dor_fetcher_client.service_url).to eq('http://127.0.0.1:3000')
-      end
+    it 'gets the configuration of the dor-fetcher client from included yml file' do
+      expect(@indexer.dor_fetcher_client.service_url).to eq('http://127.0.0.1:3000')
+    end
 
   end # ending replacing OAI context
 
-  context "whitelist" do
-    it "knows what is in the whitelist" do
+  context 'whitelist' do
+    it 'knows what is in the whitelist' do
       VCR.use_cassette('know_what_is_in_whitelist_call') do
         lambda{
-          indexer = Harvestdor::Indexer.new({:whitelist => @whitelist_path})
-          expect(indexer.whitelist).to eq(["druid:yg867hg1375", "druid:jf275fd6276", "druid:nz353cp1092"])
+          indexer = described_class.new({ whitelist: @whitelist_path })
+          expect(indexer.whitelist).to eq(['druid:yg867hg1375', 'druid:jf275fd6276', 'druid:nz353cp1092'])
         }
       end
     end
-    it "is an Array with an entry for each non-empty line in the file" do
+    it 'is an Array with an entry for each non-empty line in the file' do
       @indexer.send(:load_whitelist, @whitelist_path)
       expect(@indexer.send(:whitelist)).to be_an_instance_of(Array)
       expect(@indexer.send(:whitelist).size).to eq(3)
     end
-    it "is empty Array if there was no whitelist config setting" do
+    it 'is empty Array if there was no whitelist config setting' do
       VCR.use_cassette('empty_array_no_whitelist_config_call') do
         lambda{
-          indexer = Harvestdor::Indexer.new()
+          indexer = described_class.new
           expect(indexer.whitelist).to eq([])
         }
       end
     end
-    context "load_whitelist" do
-      it "is not called if there was no whitelist config setting" do
+    context 'load_whitelist' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.